### DISCUSSION TITLE
Cypress tests randomly fails in jenkins, unable to replicate this in local

### DISCUSSION THREAD

**Discussion by KrubaVelmurugan** [cypress-io/cypress#26210] [Questions and Help]
This is the error that I am getting while running the tests in jenkins

<img width="1121" alt="Screenshot 2023-03-24 at 11 36 48 AM" src="https://user-images.githubusercontent.com/29753136/227438677-f3a3aa2c-7f5c-4bff-a255-563245c49c6f.png">

Can someone please help?

**Comment by jjhelguero** [comment]
If you can reproduce on your local, does it give you more information?

**Reply by KrubaVelmurugan** [reply]
It is not reproducible in my local. I am facing this issue only in jenkins.
